pkg/network/ebpf/c/protocols/grpc/helpers.h

```diff
@@ -0,0 +1,130 @@
+#ifndef __GRPC_HELPERS_H
+#define __GRPC_HELPERS_H
+
+#include "bpf_builtins.h"
+
+#include "protocols/http2/decoding-defs.h"
+#include "protocols/http2/helpers.h"
+#include "protocols/grpc/defs.h"
+
+#define GRPC_MAX_FRAMES_TO_PROCESS 10
+#define GRPC_MAX_HEADERS_TO_PROCESS 10
+#define GRPC_CONTENT_TYPE_LEN 11
+#define GRPC_ENCODED_CONTENT_TYPE "\x1d\x75\xd0\x62\x0d\x26\x3d\x4c\x4d\x65\x64"
```

Comment:
Dumped it from Wireshark. Also checked it against the HPACK spec. The Huffman encoding being fixed in HPACK, this should be the only valid encoding for "application/grpc". I will add a comment explaining how this can be computed from the spec.